runtime/jcl/module.xml

```diff
@@ -68,6 +68,7 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-ex
 	<xi:include href="uma/se9_objects.xml"></xi:include>
 	<xi:include href="uma/se16_objects.xml"></xi:include>
 	<xi:include href="uma/se19_objects.xml"></xi:include>
+	<xi:include href="uma/se24exclude_objects.xml"></xi:include>
```

Comment:
This is unnecessarily complicated. Please don't change this file nor add `se24exclude_objects.xml`. Instead, update the object and export.
In `se6_vm-side_natives_objects.xml`:
```
	<object name="acccont">
		<exclude-if condition="spec.java24"/>
	</object>
```
In `se6_vm-side_natives_exports.xml`:
```
	<export name="Java_java_security_AccessController_initializeInternal">
		<exclude-if condition="spec.java24" />
	</export>
```
Also, `runtime/jcl/exports.cmake` should be updated to conditionally export that function and in `jclprots.h` the declaration should be conditional.